.github/workflows/ci-lite-test-full.yml

```diff
@@ -75,27 +79,28 @@ jobs:
           ${{ runner.os }}-pip-py${{ matrix.python-version }}-${{ matrix.release }}-${{ matrix.requires }}-
 
     - name: Install dependencies
-      env:
-        PACKAGE_NAME: pytorch  # TODO(lite) does this need to say lite?
-        FREEZE_REQUIREMENTS: 1
       run: |
         flag=$(python -c "print('--pre' if '${{matrix.release}}' == 'pre' else '')" 2>&1)
         url=$(python -c "print('test/cpu/torch_test.html' if '${{matrix.release}}' == 'pre' else 'cpu/torch_stable.html')" 2>&1)
         pip install -e .[test] --upgrade $flag --find-links "https://download.pytorch.org/whl/${url}"
         pip list
       shell: bash
 
+    - name: DocTests Lite
+      working-directory: ./src
+      run: python -m pytest lightning_lite
+
     - name: Testing Warnings
-      # the stacklevel can only be set on >=3.7
+      # the stack-level can only be set on >=3.7
       if: matrix.python-version != '3.7'
-      working-directory: tests/tests_lite
-      # needs to run outside of `pytest`
+      working-directory: ./tests/tests_lite
+      # needs to run outside `pytest`
```

Comment:
`of` sounds clearer to me, not sure if removing it is correct.

Reply:
```suggestion
      # needs to run outside of `pytest`
```